### DISCUSSION TITLE
Cannot get the access to the group

### DISCUSSION THREAD

**Discussion by micttyl** [carbon-language/carbon-lang#2142] [Getting started] [answered]
It seems I was approved to have the access, but it is somehow not allowed.
Do you know what's going on? Is this commenter system now obsolete and I did not find the update about it somewhere?

Also, I am not so familiar with Discord, so to ask, is everyone aware of that the Carbon channel(possibly entire Discord channels) requires individual's phone number to participate? You can still read chats without the verification though. Is this mandated for all?

![aaaa](https://user-images.githubusercontent.com/109993301/188303428-325c323b-bdb9-4fdb-9d4e-27aa6aecd84e.png)

![bbbb](https://user-images.githubusercontent.com/109993301/188303435-d21acab0-0a2a-4fb6-9100-09adf428dae7.png)

**Comment by jonmeow** [comment, marked as answer]
> It seems I was approved to have the access, but it is somehow not allowed. Do you know what's going on? Is this commenter system is now obsolete and I did not find the update about it somewhere?

I've changed it so that members can view conversations, but just to avoid this confusing message. There should never be conversations to view; this group is just for access control (e.g., the shared drive grants access to the group rather than individuals).

> Also, I am not so familiar with Discord, so to ask, is everyone aware of that the Carbon channel(possibly entire Discord channels) requires individual's phone number to participate? You can still read chats without the verification though. Is this mandated for all?

It's currently enabled to tamp down on some trolling we'd encountered. I expect the requirement to be [lowered to high](https://discord.com/safety/360043653152-Four-steps-to-a-super-safe-server) sometime soon, now that the surge of attention seems to have diminished.
